### ISSUE TITLE
IDEA: Here is an idea to consider

### ISSUE THREAD

**Issue by TheDevster** [bcortezf/bc_camping#1]
Instead of having to type out the command "tienda", have an item in your inventory called "Tent Kit" that can be bought from the store and you can "use" it and when you use that specific item it will invoke the same action that "tienda" does.  This way it seems more interactive.

**Comment by bcortezf** [comment]
Yeah, Im thinking of doing that, but that means that I have to adapt to a framework.

In a future I will update this in a massive way, with more features :D
